hazelcast/src/main/java/com/hazelcast/config/NearCacheConfigReadOnly.java

```diff
@@ -81,9 +81,4 @@ public NearCacheConfig setLocalUpdatePolicy(LocalUpdatePolicy localUpdatePolicy)
     public NearCacheConfig setEvictionConfig(EvictionConfig evictionConfig) {
         throw new UnsupportedOperationException("This config is read-only");
     }
-
-    @Override
-    public EvictionConfig getEvictionConfig() {
-        return super.getEvictionConfig().getAsReadOnly();
```

Comment:
Beware that now the getEvictionConfig on the NearCacheConfigReadOnly returns a mutable EvictionConfig which differs from other *ReadOnly config classes with nested instances.